src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -79,6 +79,28 @@ NetworkCommissioning::WiFiBand ToClusterObjectEnum(DeviceLayer::NetworkCommissio
     return static_cast<ClusterObject>(to_underlying(band));
 }
 
+chip::BitFlags<NetworkCommissioning::WiFiSecurity>
+ToClusterObjectBitFlags(const chip::BitFlags<DeviceLayer::NetworkCommissioning::WiFiSecurity> & security)
+{
+    using ClusterObject     = NetworkCommissioning::WiFiSecurity;
+    using PlatformInterface = NetworkCommissioning::WiFiSecurity;
```

Comment:
These are the same type, so the asserts below are trivially going to pass.

What I would suggests is that we get rid of `DeviceLayer::NetworkCommissioning::WiFiSecurity` and include `cluster-enums.h` in the relevant places so we can use `Clusters::NetworkCommissioning::WiFiSecurity` directly.

Reply:
This is discuessed before, and we want to avoid using cluster-enums in platform code.

Reply:
cluster-enums was _specifically_ created so it can be used in platform code.  Not to be confused with non-ABI-stable cluster-objects.

Reply:
+1 `cluster-enums.h` is usable in platform layer and was introduced to avoid this